Presets_Freemap-preset.xml

```diff
@@ -87,7 +87,7 @@
         <key key="amenity" value="parcel_locker"/>
         <key key="brand" value="Packeta"/>
         <key key="brand:wikidata" value="Q67809905" match="keyvalue"/>
-        <key key="colour" value="red" match="none"/>
+        <combo key="colour" text="Color" values="red,white" sk.display_values="červená,biela" match="none"/>
```

Comment:
Podľa [špecifikácie](https://wiki.openstreetmap.org/wiki/Customising_JOSM_Presets#Combobox), tag "combo" nemá atribút "match" ale "default". A hodnota atribútu "text" by mohla obsahovať slovenskú hodnotu "Farba", tak ako pri väčšine ostatných tagov. V prípade, že sa má jednať o anglickú hodnotu, tak potom prosím upraviť na anglické "Colour", ako je už uvedené v atribúte "key".

Reply:
Dakujem, ten default pozriem. Ale text je zamerne Color, aby sa to automaticky prelozilo v JOSM, ktory ma tento retazec, a nie Colour.

Reply:
Podla https://josm.openstreetmap.de/wiki/TaggingPresets moze byt 'match' aj v <combo>.

Reply:
Vdaka za odkaz, potom je to OK.

Reply:
Dakujem. Pozeral som ten 'default', ale kedze teraz mnohe nove boxy su biele, este neviem aky je v tom trend, neviem aka by mala byt default hodnota. Nechajme to zatial tak.